tasm-staff-editor/src/components/ListViewComponents/ListViewItem.js

```diff
@@ -0,0 +1,42 @@
+import Button from "../ButtonPanel/Button";
+export default function ListViewItem({ exhibit, setEntry }) {
+  if (!exhibit) {
+    console.error("ERROR: exhibit is null");
+    return null;
+  }
+  const handleDelete = () => {
+    const confirmDelete = window.confirm("Are you sure you want to delete this exhibit?");
+    if (confirmDelete) {
+      // Perform the delete operation here
+      // Still needs to be added
+      setEntry(exhibit.id);
+    }
+  };
+
+  return (
+    <tr key={exhibit.id}>
+      <td>{exhibit.data().title}</td>
+      <td>{exhibit.data().fourDigitCode}</td>
+      <td>
+        <Button
+          label="EDIT"
+          onClick={() => setEntry(exhibit.id)}
+          icon={null}
+          className="btn rounded-md py-0.5 px-3 text-xl drop-shadow-[2px_3px_4px_rgba(0,0,0,0.25)]"
+        />
+      </td>
+      <td>HIDE</td>
+      <td>ORDER</td>
+      <td>Move Up</td>
+      <td>Move Down</td>
```

Comment:
We'll need to implement this logic

One option is to switch to the 'Next' and 'Previous' drop downs, allowing us to quickly implement the functionality until the order can be setup